### DISCUSSION TITLE
2022-02-09 Weekly Meeting 42 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#178] [Weekly Meetings]
Weekly Meeting 42 is happening on Wednesday Feb 9nd, 2022 from 9:00 AM PDT - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Announcements and Updates
Housekeeping: 
- [Discuss New Working Group Template](https://github.com/omigroup/working-group-template)
  - Proposed minimum requirements for a working group: https://github.com/omigroup/omigroup/issues/176

Working Group Updates:
-  OMI Website update - Next steps? 
  - Related: OMI website group adding logos to design, related to Interoperator groups efforts. Suggest nailing down process for adding logos to website in partnership with the interoperator working group. 
- OMI-Chan working group updates! 
- WASM group
- gLTF group
- any others?

Open Floor!
 
Please leave any additional agenda items below in the comments. Thanks!

**Comment by robertlong** [comment]
I created a template repository for new working groups, based on @mrmetaverse's templates [here](https://github.com/omigroup/omigroup/issues/176)

https://github.com/omigroup/working-group-template

**Reply by mrmetaverse** [reply]
Related: continue Convo for the [minimum proposed requirements for forming a new working group](https://github.com/omigroup/omigroup/issues/176)

**Comment by RedEagleP1** [comment]
Plan to bring OCEM down in latter half to listen in.

**Comment by technobaboo** [comment]
We have some cool stuff to show from the OMI-chan working group such as our plans and concept art!

**Comment by mrmetaverse** [comment]
can we close this? https://github.com/omigroup/omigroup/issues/63 or is this separate?

**Comment by mrmetaverse** [comment]
![image](https://user-images.githubusercontent.com/63426722/153254989-4a02ef85-abb4-4a0e-bcc6-a502254e7a1a.png)
![image](https://user-images.githubusercontent.com/63426722/153255012-11125f44-f5fc-48df-bfd3-d7766a702f09.png)

As new working groups form/storm they should create a simple readme with "who we are, why we exist, how you can get involved". That [should be a minimum requirement](https://github.com/omigroup/omigroup/issues/176) so we can link to that readme from the main omigroup/omigroup working groups list. Everything else is determined at the working group level (ceremonies, working agreement, backlog process, etc.)

https://github.com/omigroup/omigroup
